sqlmesh/migrations/v0012_ensure_change_category.py

```diff
@@ -0,0 +1,24 @@
+import json
+
+
+def migrate(state_sync):  # type: ignore
```

Comment:
Why do you need this script? Aren't we doing this already as part of the `_migrate_rows`?
https://github.com/TobikoData/sqlmesh/blob/2e648f641637d3188eb6e571857d6fb3ab2b7342/sqlmesh/core/state_sync/engine_adapter.py#L842-L849